mars/services/cluster/api/web.py

```diff
@@ -161,6 +162,23 @@ async def get_node_thread_stacks(self):
             )
         )
 
+    @web_api("logs", method="get", cache_blocking=True)
+    async def get_node_log(self):
+        cluster_api = await self._get_cluster_api()
+        address = self.get_argument("address", "") or None
+        # 10MB by default
+        size = int(self.get_argument("size", str(10 * 1024 * 1024)))
```

Comment:
We can add a TODO to support chunked log later.

Reply:
Added.